Gems/Multiplayer/Code/Include/Multiplayer/IMultiplayerDebug.h

```diff
@@ -7,9 +7,177 @@
  */
 
 #pragma once
+#include <AzCore/Math/MathStringConversions.h>
 
 namespace Multiplayer
 {
+    enum class MultiplayerAuditCategory
+    {
+        MP_AUDIT_DESYNC,
+        MP_AUDIT_INPUT,
+        MP_AUDIT_EVENT
+    };
+
+    //! @class IMultiplayerAuditingDatum
+    //! @brief IMultiplayerAuditingDatum provides an interface for auditing datums of a given auditing event
+    class IMultiplayerAuditingDatum
```

Comment:
Try and move all your impls to an inline file, it's really difficult to see what your interfaces actually are (and there are no docs markup yet as well) with all the implementations mixed in together like this.

Reply:
Good call, I've cleaned this up a fair bit.